lib/driver/runtimeTool.cpp

```diff
@@ -22,6 +22,21 @@
 #include <vector>
 
 namespace WasmEdge {
+std::ostream &operator<<(std::ostream &OS, uint128_t Value) {
+  std::string S;
+  if (Value < (uint64_t)-1) {
+    OS << uint64_t(Value);
+    return OS;
+  }
+
+  while (Value != 0) {
+    S.push_back('0' + Value % 10);
```

Comment:
It's proper to separate the `uint128_t` into 2 uint64_t to print, rather than construct them into a string.